.github/opensearch-cluster/docker-compose.yml

```diff
@@ -8,6 +8,7 @@ services:
       - '9600:9600'
     environment:
       - 'OPENSEARCH_INITIAL_ADMIN_PASSWORD=${OPENSEARCH_PASSWORD:-myStrongPassword123!}'
+      - 'OPENSEARCH_JAVA_OPTS=${OPENSEARCH_EXPERIMENTAL_FEATURES}'
```

Comment:
The list of opts is not a list of experimental features, so this should be `OPENSEARCH_JAVA_OPTS=${OPENSEARCH_JAVA_OPTS}`.

Reply:
updated